package.json

```diff
@@ -66,7 +66,7 @@
     "uuid": "^9.0.0"
   },
   "engines": {
-    "node": ">=10.0.0"
+    "node": "22.11.x"
```

Comment:
Is this a good idea? I think this means our package won't advertise itself as compatible with other versions of node, which is probably right given that we do not test it with other versions of node?